### DISCUSSION TITLE
根据已有标签决定是否添加新的标签[add tags according to current tags]

### DISCUSSION THREAD

**Discussion by LamFTS** [windingwind/zotero-actions-tags#270] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

- OS: Archlinux
- Zotero Version: Zotero Beta 55
- Plugin Version: 1.0.0-beta.33

### Describe the feature request

我想根据当前条目已有的标签决定现有标签如何添加，比如,我想设置三个标签状态，分别是`/unread`、`/open`和`/readed`，我希望打开条目时自动添加`/opened`标签并自动移除`/unread`，但是如果此条目已经被赋予了`/readed`标签，则再次打开该条目不执行添加`/open`标签。以前可以通过设置：`'/open[!/readed]',~~/unread`实现，但是新版本貌似是需要自定义脚本，希望能出一个教程或脚本。非常感谢。
I want to add tags based on the existing tags for the current item. For example, I want to set three tag states, namely `/unread`, `/open`, and `/readed`. I want to automatically add the `/open` tag and remove the `/unread` tag when opening item, but if the item has already been assigned the `/readed` tag, opening it again will not perform addition of `/open` tag. Previously, it was possible to achieve this by setting: `'/open[!/read]',~~/unread'`, but the new version seems to require custom scripts. We hope to have a tutorial or script available. Thanks.

### Describe the solution you'd like

如上述[described above]。

### Anything else?

类似的，我想设置一个动作，当自动添加`/open`标签时，移除`/unread`标签该如何操作呢？
Similarly, I would like to set an action to remove the `/unread` tag when `/open` is automatically added.

**Comment by windingwind** [comment]
可参考https://github.com/windingwind/zotero-actions-tags/discussions/202

**Comment by syuuraku** [comment]
我也很需要这个功能，请问现在解决了吗？

**Comment by syuuraku** [comment]
I have written a script for this, and please refer to #423.
